### PR TITLE
storeapi: reduce the mccabe complexity

### PR DESCRIPTION
**Pull request by elopio**
LP: #1541011

### CODE REVIEW

**Comment by elopio** [comment]
Next, replace all the static checks with flake8.

File: snapcraft/storeapi/_upload.py [hunk 1]
```diff
@@ -267,6 +207,71 @@ def upload_app(name, upload_data, metadata=None, config=None):
...
+            ]
+            status_web_url = response_data.get('web_status_url')
+            if status_web_url:
+                result['errors'].append(
+                    'Please check the status later at: %s.' % (
+                        status_web_url),
+                )
+        result['application_url'] = data.get('application_url', '')
+    else:
+        logger.error(
```
Comment (sergiusens) [inline snapcraft/storeapi/_upload.py]: why not take the opportunity to use `.format` :wink:

Comment (elopio) [inline snapcraft/storeapi/_upload.py]: Done.

**Comment by sergiusens** [comment]
:+1:

**Pull request merged**